llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1468,15 +1471,36 @@ def FeatureISAVersion9_4_Common : FeatureSet<
 
 def FeatureISAVersion9_4_0 : FeatureSet<
   !listconcat(FeatureISAVersion9_4_Common.Features,
-    [FeatureForceStoreSC0SC1])>;
+    [
+      FeatureForceStoreSC0SC1,
+      FeatureFP8Insts,
```

Comment:
IIUC, we don't use `FeatureFP8Insts` anywhere else. We don't guard anything with `HasFP8Insts`. Not sure if that is there just for conventions.